facturacion_mexico/cfdi_recibidos/queries.py

```
@@ -0,0 +1,98 @@
+			JOIN `tabItem Group` ig ON ig.name = i.item_group
+			WHERE i.is_purchase_item = 1
+			  AND i.is_stock_item = 0
+			  AND i.is_sales_item = 0
+			  AND ig.is_group = 0
+			  AND ig.lft > %s
+			  AND ig.rgt < %s
+			  AND i.stock_uom IN ({sat_placeholders})
+			  AND (i.name LIKE %s OR i.item_name LIKE %s)
+			ORDER BY i.name
+			LIMIT %s, %s
+			""",
```

Ruff (0.15.14)

[error] Possible SQL injection vector through string-based query construction

(S608)